README.md

```diff
@@ -41,7 +41,7 @@ Then, create objects on your index:
 client = Algolia::Search::Client.create('YourApplicationID', 'YourAPIKey')
 index = client.init_index('your_index_name')
 
-index.save_objects([objectID: 1, name: 'Foo'])
+index.save_objects({objectID: 1, name: 'Foo'})
```

Comment:
```suggestion
index.save_objects([{objectID: 1, name: 'Foo'}])
```

Reply:
Oh my god, I went too quickly and now I understand why it was working!!